test/parallel/test-buffer-fill.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
 const assert = require('assert');
+const errors = require('../../lib/internal/errors');// Flags: --expose-internals
```

Comment:
I think the comment needs to be the first line of the file. The idea of using it is to be able to change `../../lib/internal/errors` on this line to `internal/errors`. This way, the test runs based on version of node that is compiled and not the contents of `lib` in the source tree at the time the test is run.